protocol_tests/aws-restjson/protocols/Aws_restJson1_1.ts

```diff
@@ -1536,7 +1536,7 @@ const deserializeAws_restJson1_1GreetingWithErrorsCommandError = async (
   errorCode = loadRestJsonErrorCode(output, parsedOutput.body);
   switch (errorCode) {
     case "ComplexError":
-    case "aws.protocols.tests.restjson#ComplexError":
```

Comment:
pretty sure this file should be deleted in favor of `Aws_restJon1.ts`, which was generated

Reply:
Nice catch, removed.